client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx

```diff
@@ -11,15 +11,20 @@ import styles from './BatchChangesListIntro.module.scss'
 export const BatchChangesChangelogAlert: React.FunctionComponent<React.PropsWithChildren<unknown>> = () => (
     <DismissibleAlert
         className={styles.batchChangesListIntroAlert}
-        partialStorageKey="batch-changes-list-intro-changelog-3.39"
+        partialStorageKey="batch-changes-list-intro-changelog-3.40"
     >
         <Card className={classNames(styles.batchChangesListIntroCard, 'h-100')}>
             <CardBody>
-                <Typography.H4 as={Typography.H2}>Batch Changes updates in version 3.39</Typography.H4>
+                <Typography.H4 as={Typography.H2}>Batch Changes updates in version 3.40</Typography.H4>
                 <ul className="mb-0 pl-3">
                     <li>
-                        Bulk actions are now visible regardless of filtering. Previously, you had to filter by status to
-                        see them.
+                        <Link
+                            to="https://docs.sourcegraph.com/batch_changes/explanations/introduction_to_batch_changes#supported-code-hosts-and-changeset-types"
+                            rel="noopener"
+                            target="_blank"
+                        >
+                            Bitbucket Cloud is now supported with Batch Changes
```

Comment:
```suggestion
                            Bitbucket Cloud is now supported with Batch Changes.
```

(don't we usually punctuate this? 😅 idk if it matters)

Reply:
I did that first but didn't really like the looks of it 😆 I can change it though?